README.md

```diff
@@ -40,8 +40,12 @@ Start libvirtd if needed:
 systemctl start libvirtd.service
 ```
 
-It is possible that your Linux distribution don't configure your user for libvirt use by default.
-If it's not the case, you should configure it following your distribution recommendations.
+We then need to allow the developer (an unprivileged user) to use libvirt thanks to a dedicated group:
```

Comment:
I tested this on Debian and Arch Linux. Is it different on Fedora?

Reply:
I may be wrong but if understand it well the `unix_sock_group` configuration is used for libvirt communication if polkit is not configured on the system. This is in my knowledge not the case of fedora, debian or archlinux. Thus I think it's ok to avoid this configuration.

Reply:
Sorry, I was talking about my first configuration recommendation `unix_sock_group = "libvirt"`. Asking users to add themself to the `libvirt` group as you wrote should be enough for major usecases.